src/components/map/MapLoadingMask.js

```diff
@@ -1,16 +1,10 @@
 import { ComponentCover, CenteredContent, CircularLoader } from '@dhis2/ui'
-import cx from 'classnames'
 import React from 'react'
-import styles from './styles/MapLoadingMask.module.css'
 
 export const loadingMaskClass = 'dhis2-map-loading-mask'
 
 const MapLoadingMask = () => (
-    <ComponentCover
-        translucent
-        className={cx(styles.cover, loadingMaskClass)}
-        dataTest="map-loading-mask"
-    >
+    <ComponentCover className={loadingMaskClass} dataTest="map-loading-mask">
```

Comment:
Was removal of `translucent` to fix something else, or was it for these plugin changes?

Reply:
This is to avoid the (one) flashing grey background.